pytorch/src/main/java/org/bytedeco/pytorch/presets/torch.java

```diff
@@ -1824,6 +1930,7 @@ We need either to put an annotation info on each member, or javaName("@NoOffset
                .put(new Info("torch::jit::Node").pointerTypes("JitNode"))
                .put(new Info("torch::jit::Module").pointerTypes("JitModule"))
                .put(new Info("torch::jit::Object").pointerTypes("JitObject"))
+               .put(new Info("torch::jit::load").javaNames("jitLoad"))
                .put(new Info("torch::jit::String").pointerTypes("JitString"))
```

Comment:
Please remove the jit prefixes from all those names. Like I keep telling you, don't change anything that doesn't need to be changed.

Reply:
All those symbols from `torch::jit` module should really be in a separate Java package, since the related features are not needed by most users, but we cannot do this.
`JitModule` and `JitNode` are needed since we already other classes named `Module` and `Node` in the pytorch package.
`JitString` and `JitObject` are for convenience I guess. Anyway all those are not changed in 2.1.0, they were already mapped as such in your versions of the presets.

`jitLoad` is new and is a mistake. I though there were a conflicting `load` method but I cannot find it any more. I'll revert that.